README.md

```diff
@@ -61,6 +61,10 @@ export default function Home() {
 }
 ```
 
+### [SSR] Inline chart example preview
+
+![](./images/preview1.svg)
```

Comment:
very nice, didn't know you could use repository-specific paths. Will definitely keep this in mind. Thanks!